src/components/buttons/accessible-button/README.md

```diff
@@ -39,19 +39,19 @@ export default Button;
-| buttonAttributes | Object   |     -     | Attributes to be applied to the `<button/>` element. Can be use to e.g. add data-\* attributes to the button. |
+| Property         | Type      | Required? | Description                                                                                                                                                                                  |
+| :--------------- | :-------- | :-------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id               | String    |     -     | Used as the HTML `id` attribute.                                                                                                                                                             |
+| type             | String    |     -     | Used as the HTML `type` attribute. Supported values are `submit`, `reset` and `button`.                                                                                                      |
+| label            | String    |     ✓     | The text that should be used as `aria-label`                                                                                                                                                 |
+| children         | Node      |     ✓     | Button content                                                                                                                                                                               |
+| isToggled        | Bool      |     -     | Set this prop to indicate a toggle button and it's state                                                                                                                                     |
+| ref              | React ref |     -     | Used as button reference                                                                                                                                                                     |
+| isToggleButton   | Bool      |     -     | Used to set `aria-pressed˚ state                                                                                                                                                             |
+| isDisabled       | Bool      |     -     | Set this prop to indicate a disabled button                                                                                                                                                  |
+| onClick          | Function  |     ✓     | Callback that is called when the button is clicked                                                                                                                                           |
+| className        | String    |     -     | Will be applied to the `<button/>` to allow for custom styling. Alternatively, you can pass a `css` prop if you're [styling your components with Emotion](https://emotion.sh/docs/css-prop). |
```

Comment:
I'm not sure we need to include this doc because it's pretty much just explaining how emotion `css` prop works 🤔

Reply:
Yeah I'll remove it, also because passing a `className` doesn't really work. See https://github.com/commercetools/ui-kit/pull/459#issuecomment-456497484